jdaviz/tests/test_show.py

```diff
@@ -0,0 +1,21 @@
+from jdaviz import Specviz
+import sidecar
+
+
+def test_show_inline():
```

Comment:
Use pre-defined fixtures in `conftest.py`.
```suggestion
def test_show_inline(specviz_app):
```

Reply:
Thanks, did not know about the fixtures!